### PR TITLE
moving deep learning to example-dev private repo

### PR DESCRIPTION
**Pull request by alifarhid**

### CODE REVIEW

**Review by hadia206 — APPROVED** [review]
thanks @alifarhid

File: README.md [hunk 1]
```diff
@@ -20,17 +20,14 @@ By default all examples and data generation scripts can be run from home directo
...
-- [Intraday Mean](https://github.com/Bodo-inc/Bodo-examples/blob/master/examples/intraday_mean.py)
-  - [data generation](https://github.com/Bodo-inc/Bodo-examples/blob/master/data/stock_data_read.py)
-- [TPCH Queries](https://github.com/Bodo-inc/Bodo-examples/tree/master/examples/tpch), more information on TPC-H can be found [here](http://www.tpc.org/tpch/)
-  - data generation: generated data will be available at `data/tpch-datagen/data`:
-
-		# To generate data with a scale of 1, equivalent to 1GB of data
-		data/tpch-datagen/generateData.sh 1
-		# To generate data with a scale of 2, equivalent to 2GB of data
-		data/tpch-datagen/generateData.sh 2
+- [Some TPCH Queries](https://github.com/Bodo-inc/Bodo-examples/tree/master/examples/tpch), more information on TPC-H can be found [here](http://www.tpc.org/tpch/)
```
Comment (ehsantn) [inline README.md:23]: I think we have all TPC-H Queries, not just some.

Comment (hadia206) [reply README.md:23]: Agree. I update the README a month ago.
https://github.com/Bodo-inc/Bodo-examples/commit/5589f0b4e211ed33ab6827ded41222768137754d
@alifarhid this is an old README file. Please use the latest one.

Comment (alifarhid) [reply README.md:23]: Got it @ehsantn and @hadia206. Fixed it. Thanks.

**Pull request merged**